### PR TITLE
Change k8s default version to 1.8.9

### PR DESCRIPTION
**Pull request by CecileRobertMichon**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by CecileRobertMichon** [comment]
@dmitsh for review/validation. cc: @jackfrancis

File: examples/k8s-upgrade/v1.7.9-hybrid.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "properties": {
     "orchestratorProfile": {
       "orchestratorType": "Kubernetes",
-      "orchestratorVersion": "1.7.9"
```
Comment (dmitsh) [inline examples/k8s-upgrade/v1.7.9-hybrid.json]: Do we drop windows support for 1.7? If so, please rename the file to examples/k8s-upgrade/v1.8.8-hybrid.json and update test config test/acse-conf/acse-upgrade-win.json

File: examples/k8s-upgrade/v1.7.9-win.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "properties": {
     "orchestratorProfile": {
       "orchestratorType": "Kubernetes",
-      "orchestratorVersion": "1.7.9"
```
Comment (dmitsh) [inline examples/k8s-upgrade/v1.7.9-win.json]: same here

File: examples/k8s-upgrade/v1.7.9-hybrid.json (hunk 1 above)
Comment (CecileRobertMichon) [reply examples/k8s-upgrade/v1.7.9-hybrid.json]: I didn't realize this was testing upgrade, I reverted the change. Thanks!

**Comment by jackfrancis** [comment]
We'll wait to do this until we have Windows support for v1.8.9 (see unit tests)

**Pull request merged**